dash_docs/chapters/react_for_python_developers/index.py

```diff
@@ -143,9 +143,8 @@
 ### Build the project
 
 - `npm run build:js` generate the production bundle `project_shortname.min.js`
```

Comment:
```suggestion
- `npm run build:js` generate the JavaScript bundle `project_shortname.min.js`
```
Now that there aren't separate prod/dev bundles - if you want we could mention the `.js.map` file but that's probably not important enough for this part.